index.css

```diff
@@ -0,0 +1,93 @@
+.body{
+    display: flex;
+    flex-direction: column;
+    min-width: 100vw;
+    min-height: 100vh;
+}
+.background-top{
+    display: flex;
+    width: 100%;
+    height: 50%;
+    flex-direction: row;
+    background: rgba(128, 0, 128, 0.75);
+    justify-content: space-between;
+}   
+.background-bottom{
+    display: flex;
+    width: 100%;
+    height: 50%;
+    
+    
+    background: #D9D9D9;
+}
+.background-center{
+    display: flex;
+    width: 100%;
+    height: 10%;
+    flex-direction: row;
+    
+}
+.search-bar{
+    display: flex;
+    width: 96%;
+    height: 50px;
+    position: relative;
+    margin: 2%;
+    background: #FFFFFF;
+}
+
+.pinkBall-01{
+    display: flex;
+    height: 100px;
+    width: 76px;
+    border-radius: 34px 265px 321px 30px;
+    background: rgba(211, 31, 139, 0.75);
+}
+.pinkBall-02{
+    display: flex;
+    height: 100px;
+    width: 76px;
+    border-radius: 265px 34px 30px 321px;
+   
+   
+    
+    background: rgba(211, 31, 139, 0.75);
+}
+.center-line{
+    display: flex;
+    width: 100%;
+    height: 100%;
+    background:#555555;
+    align-items: center;
+}
+.center-circle{
+    display: flex;
+    width: 133px;
+    height: 143px;
+    left: 42%;
+    position: absolute;
+    border-radius: 50%;
+    
+    background: #555555;
+}
+
+.master-text{
+    position: absolute;
+width: 261px;
+height: 62px;
+left: 34%;
+top: 54px;
+
+font-family: 'Risque';
+font-style: normal;
+font-weight: 400;
+font-size: 55px;
+line-height: 42px;
+
+color: #FFFFFF;
+
+}
+
+.input{
```

Comment:
I don't see a need for any input here.  If it's not necessary please do not include it in this ticket.